substrate/frame/revive/src/exec.rs

```diff
@@ -1073,6 +1082,7 @@ where
 					&frame.account_id,
 					frame.contract_info.get(&frame.account_id),
 					executable.code_info(),
+					self.skip_transfer,
```

Comment:
Don't we need to mock the normal balance transfer further down in this function, too ( `Self::transfer_from_origin` ) ?

Reply:
No this one is fine, if you explicitly specify a value to transfer then your call should fail

https://github.com/paritytech/polkadot-sdk/blob/76e3fdaa26fdfec1d2ac71e25649f140f55d2156/substrate/frame/revive/rpc/examples/js/src/geth-diff.test.ts?plain=1#L132-L150